fern/products/sdks/overview/go/publishing-to-go-package-manager.mdx

```diff
@@ -1,24 +1,23 @@
 ---
 title: Publishing to Pkgsite
-description: How to publish the Fern Go SDK to Pkgsite.
+description: How to publish the Fern Go SDK to the Go module proxy.
 ---
 
-Publish your public-facing Fern GO SDK to the [Pkgsite
-registry](https://pkg.go.dev/). After following the steps on this page,
-you'll have a versioned package published on Pkgsite.
+Publish your public-facing Fern Go SDK to the [Go module
+proxy](https://pkg.go.dev/). After following the steps on this page, you'll have
+a versioned package published on pkg.go.dev.
```

Comment:
I'd continue to use `pkg.go.dev` here though (w/ a link) since that still captures the main idea.